code/datums/mind.dm

```diff
@@ -256,7 +256,7 @@
 	remove_rev()
 	SSticker.mode.update_cult_icons_removed(src)
 
-/datum/mind/proc/equip_traitor(employer = "The Syndicate", silent = FALSE, datum/antagonist/uplink_owner)
+/datum/mind/proc/equip_traitor(employer = "You, criminal elements, rival factions, or", silent = FALSE, datum/antagonist/uplink_owner)
```

Comment:
```suggestion
/datum/mind/proc/equip_traitor(employer = "You, criminal elements, rival factions, or the Syndicate", silent = FALSE, datum/antagonist/uplink_owner)
```
Otherwise this will read as "You, criminal elements, rival factions, or have cunningly disguised..."

Reply:
Actually I was wrong. This variable doesn't matter **at all** and the game instead checks the one set in datum_traitor.dm.

Evidence: https://cdn.discordapp.com/attachments/607755178304733194/688225929616818235/neato.png